README.md

```diff
@@ -14,7 +14,7 @@
 - Verify if there are any memory leaks using Visual leak detector
 
 ### Checklist for Advanced Tasks / TODO
-- Use a Gitwork flow
+- Use a Git work-flow
 - Use Git actions 
```

Comment:
Thanks for the correction. Can you please also add more information to the readme?